src/ua/data_types/enum_definition.rs

```diff
@@ -0,0 +1,13 @@
+use crate::ua;
+
+use super::EnumField;
+
+crate::data_type!(EnumDefinition);
+
+impl EnumDefinition {
+    #[must_use]
+    pub fn fields(&self) -> Option<ua::Array<EnumField>> {
```

Comment:
@sgoll I would prefer to use a `get_`-Prefix for all those methods. Then non-owned variants could be introduced later without breaking changes.